Sources/SwiftyXPC/XPCFileDescriptor.swift

```diff
@@ -7,10 +7,17 @@
 
 import Darwin
```

Comment:
We will need to import `System` in order to use `FileDescriptor`.

```suggestion
import Darwin
import System
```